drivers/hmis/app/graphql/types/hmis_schema/current_living_situation.rb

```diff
@@ -40,5 +41,17 @@ def enrollment
     def client
       load_ar_association(object, :client)
     end
+
```

Comment:
Hm, so of course, I wasn't quite right about my assertion on our call earlier that our existing form behavior will just work, but please let me know what you think about this hack and commentary.

Reply:
I think this is makes sense, and thanks for the explanatory comment!